src/models/transactions/offerCancel.ts

```diff
@@ -0,0 +1,21 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+export interface OfferCancel extends BaseTransaction {
+    TransactionType: "OfferCancel";
+    OfferSequence: number;
+}
+
+/**
+ * Verify the form and type of an OfferCancel at runtime.
+ * 
+ * @param tx - An OfferCancel Transaction
+ * @returns - Void.
+ * @throws - When the OfferCancel is Malformed.
+ */
+ export function verifyOfferCancel(tx: OfferCancel): void {
+    verifyBaseTransaction(tx)
+
+    if (tx.OfferSequence !== undefined && typeof tx.OfferSequence !== 'number')
```

Comment:
I think if OfferSequence is undefined this should actually throw an error (I don't think the field is optional - https://xrpl.org/offercancel.html)

Reply:
Done!

Reply:
For `tx.OfferSequence` since it's not optional I think the condition is actually:
`(typeof tx.OfferSequence !== 'number')`

Because OfferSequence isn't optional, we want this to throw an error when OfferSequence is undefined (And since typeof undefined !== number, that's the only condition we need)